pylint/checkers/base/basic_checker.py

```diff
@@ -731,7 +753,9 @@ def leave_tryfinally(self, _: nodes.TryFinally) -> None:
         self._tryfinallys.pop()
 
     def _check_unreachable(
-        self, node: nodes.Return | nodes.Continue | nodes.Break | nodes.Raise
+        self,
+        node: nodes.Return | nodes.Continue | nodes.Break | nodes.Raise,
```

Comment:
```suggestion
        node: nodes.Return | nodes.Continue | nodes.Break | nodes.Raise | nodes.Call,
```

Since visit_call is also using this fn.